persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/models/Converter.java

```diff
@@ -35,10 +35,10 @@ public interface Converter<T> {
   T fromResultSet(ResultSet rs) throws SQLException;
 
   /**
-   * Convert a model into a Map with keys as snake case names, where as values as values of member
-   * of model obj.
+   * Convert a model into a Map with keys as snake case names, and values as values of member of
+   * model obj.
    */
-  Map<String, Object> toMap(DatabaseType databaseType);
+  Map<String, Object> toMap(DatabaseType databaseType, int schemaVersion);
```

Comment:
We don't have to change this interface if we allow the `ModelEntity` carry a new field for schema version.

Reply:
i agree, though this would mean carrying a derived column in the model considering this i added this to the signature of this interface, please let me know if you feel strongly about it.